pkg/accounting/db.go

```diff
@@ -46,30 +44,34 @@ type Rollup struct {
 	AtRestTotal    float64
 }
 
-// DB stores information about bandwidth and storage usage
-type DB interface {
-	// LastTimestamp records the latest last tallied time.
-	LastTimestamp(ctx context.Context, timestampType string) (time.Time, error)
-	// SaveAtRestRaw records raw tallies of at-rest-data.
-	SaveAtRestRaw(ctx context.Context, latestTally time.Time, created time.Time, nodeData map[storj.NodeID]float64) error
-	// GetRaw retrieves all raw tallies
-	GetRaw(ctx context.Context) ([]*Raw, error)
-	// GetRawSince retrieves all raw tallies since latestRollup
-	GetRawSince(ctx context.Context, latestRollup time.Time) ([]*Raw, error)
-	// GetStoragenodeBandwidthSince retrieves all storagenode_bandwidth_rollup entires since latestRollup
-	GetStoragenodeBandwidthSince(ctx context.Context, latestRollup time.Time) ([]*StoragenodeBandwidthRollup, error)
-	// SaveRollup records raw tallies of at rest data to the database
+// StoragenodeAccounting stores information about bandwidth and storage usage for storage nodes
+type StoragenodeAccounting interface {
```

Comment:
just an idea,  could we call this `StoragenodeDB`, then since its in the accounting package it would be referred to as `accounting.StoragenodeDB`.  Though Im not sure if calling it `Storagenode` would be confusing and not descriptive enough, rather im just brainstorming how to make these names shorter.

Reply:
That's a good point, I like StoragenodeDB.

Reply:
That name would make me think it was meant to be used _by_ a storage node.

Reply:
oh true...